src/PKSim.Core/Mappers/BoxWhiskerChartDataToDataTableMapper.cs

```diff
@@ -33,16 +33,18 @@ protected override IEnumerable<DataRow> AddSpecificChartValues(DataRow row, Curv
          for (var i = 0; i < curveData.XValues.Count; i++)
          {
             var newRow = row.Table.NewRow();
+            var yValue = curveData.YValues[i];
             newRow.ItemArray = row.ItemArray;
             newRow[_xValue] = curveData.XValues[i].ToString(curveData.XAxis);
-            newRow[_lowerWhisker] = ValueForDataTableFor(curveData.YAxis, curveData.YValues[i].LowerWhisker);
-            newRow[_lowerBox] = ValueForDataTableFor(curveData.YAxis, curveData.YValues[i].LowerBox);
-            newRow[_median] = ValueForDataTableFor(curveData.YAxis, curveData.YValues[i].Median);
```

Comment:
this was the error everywhere. Using the dimension of the Y axis instead of the YDimensiion defined at the curve level (was not easy to find!)